src/celldega/qc/__init__.py

```diff
@@ -174,12 +126,13 @@ def qc_segmentation(transcript_metadata_file, transcript_data_file, cell_polygon
         "assigned_transcripts": (trx_meta.groupby(gene)[transcript_index].count() / trx.groupby(gene)[transcript_index].count()).fillna(0)
     })
 
-    metrics_df.to_csv(path_output_cell_metrics)
-    gene_specific_metrics_df.to_csv(path_output_gene_metrics)
+    metrics_df.to_csv(os.path.join(base_path, "cell_specific_qc.csv"))
+    gene_specific_metrics_df.to_csv(os.path.join(base_path, "gene_specific_qc.csv"))
 
     print("segmentation metrics calculation completed")
 
-def mixed_expression_calc(default_segmentation_segmentation_name, default_segmentation_cell_feature_matrix_path, algorithm_names, algorithm_specific_cbg_files, cell_type_A_specific_genes, cell_type_B_specific_genes, cell_A_name, cell_B_name, cmap='cividis'):
+def mixed_expression_calc(base_paths, cell_type_A_specific_genes, 
```

Comment:
Please consider renaming the function to `orthogonal_expression_calc()` or something  else that is more informative as we mentioned earlier.